builder/azure/arm/config.go

```diff
@@ -87,11 +87,11 @@ type SharedImageGallery struct {
 	// regions where you are deploying.
 	ImageVersion string `mapstructure:"image_version" required:"false"`
 
-	// Id of the community gallery image : /CommunityGalleries/{cgName}/Images/{img}[/Versions/{}] (Versions part is optional)
-	CommunityGalleryImageId string `mapstructure:"communityGallery_image_id" required:"false"`
+	// Id of the community gallery image : /CommunityGalleries/{galleryUniqueName}/Images/{img}[/Versions/{}] (Versions part is optional)
+	CommunityGalleryImageId string `mapstructure:"community_gallery_image_id" required:"false"`
 
-	// Id of the direct shared gallery image : /sharedGalleries/{cgName}/Images/{img}[/Versions/{}] (Versions part is optional)
-	SharedGalleryImageID string `mapstructure:"directSharedGallery_image_id" required:"false"`
+	// Id of the direct shared gallery image : /sharedGalleries/{galleryUniqueName}/Images/{img}[/Versions/{}] (Versions part is optional)
```

Comment:
If an image version is provided in the community or direct shared gallery image does the value of ImageVersion get used? 

Do they both need to be identical in order for things to work?

Reply:
I went ahead and updated the documentation to include these new configuration. I think there are a few inconsistencies that we should fast follow on. But I did want to get the documentation in a better place for now.